CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ _None_
 
 - Update the CI image used to build this project to use `xcode-14.1`. [#431]
 
+## 6.2.0
+
+### New Features
+
+- Add a `is_draft` parameter to the `create_release` action to specify whether the release should be created as a draft.
```

Comment:
💄  Nit: As you can see in the other entries for inspiration, we like to include the PR number in brackets for each entry, to make it easier to find and go read the details of what the changes for that entry were about if one wants to.

```suggestion
- Add a `is_draft` parameter to the `create_release` action to specify whether the release should be created as a draft. [#433]
```

Reply:
Address in https://github.com/wordpress-mobile/release-toolkit/pull/433/commits/30fd7b4fb27ec951a10e3bb6c37bbee9bd777e9e